docs/account-management/auth/connect-to-gdn.md

```diff
@@ -60,11 +62,21 @@ main()
 
 ```py
 from c8 import C8Client
+
+# Define constants
+URL = "gdn.paas.macrometa.io"
+GEO_FABRIC = "_system" # Change this if you need to log in to a fabric other than _system.
+API_KEY = "my API key" # Change this to your API key.
+
+# Authenticate with your email and password
 client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443,
                     email='nemo@nautilus.com', password='xxxxx',
                     geofabric='_system')
 
-# Or using token
+# Authenticate with API key (recommended for apps)
+client = C8Client(protocol='https', host=URL, port=443, apikey = API_KEY, geofabric = GEO_FABRIC)
+
+# Authenticate with JWT
 client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443, token="XXXX")
```

Comment:
since now that we have defined the variable URL here so instead of gdn.paas.macrometa.io we should use that variable and maybe you can also the fabric too, i.e. you can add `geofabric=GEO_FABRIC` here too just like in the above line